HyPDF.text.rb

```diff
@@ -0,0 +1,7 @@
+File.open('public/down-the-rabbit-hole.txt', 'w:UTF-8') {|f| f.write(@hypdf = HyPDF.pdftotext(
+      Rails.root + 'public/down-the-rabbit-hole.pdf',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.